### PR TITLE
Add keep alive step to GHA

### LINKED ISSUE #2: workflows disabled if repository has no activity in 60 days
image

Could prevent with something like this https://github.com/gautamkrishnar/keepalive-workflow ...

### PR DESCRIPTION
**Pull request by jrbourbeau**
Closes https://github.com/pangeo-bot/dispatcher/issues/2

cc @scottyhq

### CODE REVIEW

File: .github/workflows/watch-pangeo-dask-feedstock.yml [hunk 1]
```diff
@@ -106,3 +106,9 @@ jobs:
                 -  distributed =`${{ steps.latest_version_distributed.outputs.version }}`
                 -  dask-kubernetes =`${{ steps.latest_version_k8s.outputs.version }}`
                 -  dask-gateway =`${{ steps.latest_version_gateway.outputs.version }}`
+
+      - name: Keep scheduled jobs alive
+        uses: gautamkrishnar/keepalive-workflow@1.0.5
+        with:
+          committer_username: "github-actions[bot]"
+          committer_email: "41898282+github-actions[bot]@users.noreply.github.com"
```
Comment (jrbourbeau) [inline .github/workflows/watch-pangeo-dask-feedstock.yml:114]: Note this should make commits appear to be from the GitHub Actions bot similar to this commit https://github.com/dask/dask-docker/commit/1df1c32d664fcfeff018b5aa323140abc4371bd1

**Comment by scottyhq** [comment]
thanks @jrbourbeau , looks good to me, definitely worth a try!

**Comment by jrbourbeau** [comment]
Thanks for reviewing @scottyhq! Let's give it a try

**Comment by scottyhq** [comment]
Didn't seem to work unfortunately b/c the actions are once again disabled with "This scheduled workflow is disabled because there hasn't been activity in this repository for at least 60 days. Enable this workflow to resume scheduled runs." I'll go ahead and re-enable them manually

**Pull request merged**